packages/vscode-extension/src/controls/sampleDetailPage.tsx

```diff
@@ -66,6 +71,16 @@ export default class SampleDetailPage extends React.Component<SampleDetailProps,
   };
 
   onViewGithub = () => {
+    vscode.postMessage({
```

Comment:
Could we send telemetry in OpenExternalLink handler?

Reply:
I prefer to send it in business code where util API is called. The openUrl API is just an util function without any context. Many of the telemetry events are already sent in places where the API is called like in openDocumentHandler(), 'TelemetryEvent.Documentation' telemetry event is sent with property TelemetryProperty.DocumentationName: "general" attached. So I suppose that's better practice.